ui/js/controllers/perf/graphs.js

```diff
@@ -485,8 +491,16 @@ perf.controller('GraphsCtrl', [
                     zoomGraph();
                 });
 
-                //Closes popup on-click of "X"
-                $('#close-popup').bind("click", plotUnselected);
+                // Close popup onclick of corner close button
+                $('#close-popup').bind("click", closePopup);
+
+                // Close pop up when user clicks outside of the graph area
+                $('html').click(closePopup);
+
+                // Stop propagation when user clicks inside the graph area
+                $('#graph').click(function(event){
```

Comment:
Could you put a space between `(event)` and `{`?

This works great aside from that. :)